en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+
+You should assign a switch on your RC controller for changing between the multicopter and fixed wing modes.
+
+:::note
+While PX4 allows flight without an RC controller, you must have one when tuning/configuring a new airframe.
+:::
+
+This is done in [Flight Mode](../config/flight_mode.md) configuration, where you [assign flight modes and other functions](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to switches on your RC controller.
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+MC-mode tuning is dificult because some axes depend on control surfaces, while others depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+As a sanity check, we start off by setting the following gain parameters:
+
+- Roll rate control:
+  - [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = `0`, 
+  - [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = `1`
+- Pitch rate control
+  - [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = `0`
+  - [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = `1`
+- Yaw rate control
+  - [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P) = [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I) = [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D) = `0`
+  - [MC_YAWRATE_K](../advanced_config/parameter_reference.md#MC_YAWRATE_K) = `1`
+- Angle control
+  - [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) = [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) = [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P) = `0`
+
+The expected behaviour is that when you arm in [Stabilized mode](../flight_modes/manual_stabilized_mc.md), both the motors should spin, control surfaces should be centred, and no effect of manual RC input using the transmistter is observed.
+
+:::note IF you can do so safely!
+Gently pick up the vehicle by hand and try tilting it in different axes.
+You should feel no countering torque from the motors or control surfaces.
+:::
+
+Once this sanity check is complete, we move on to slowly increasing the gains.
+
+### Rate Gains
+
+- Increase [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) by 0.1.
+  Arm and lift the vehicle and displace it in the roll axis.
```

Comment:
I assume this is the roll axis of the vehicle in MC mode? i.e. hovering? And the the direction of roll is relative to the nominal front (as inferred from the vehicle in FW mode)?

What does displace it mean. If the vehicle was hovering, I assume this just means pushing the bottom from side to side "against the axis of FW flight"?

Reply:
So, in MC Mode, the Roll axis is the one being actuated by differential thrust. So displacing it here means pushing one of the sides down and letting it go.

Reply:
I'm envisaging something that is hovering using tail motors. So it kind of has "no sides" - or at least in the long axis. Sorry if I am being dim. I think I need a picture.

Reply:
I agree it's a little confusing. I'll add a diagram of what I mean by displacing in the axes as well.

Reply:
Thank you!